src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java

```diff
@@ -565,5 +567,18 @@ public static void clearPostHearingRequestFormatAndContentFields(SscsCaseData ca
             }
         }
     }
+
+    public static boolean isIbcaCase(SscsCaseData caseData) {
```

Comment:
see other comment, maybe this could be a method on the selected benefit type?

Reply:
this wouldn't work with the previous one, as the other selected benefit type is using a SYA specific SyaBenefitType class, whereas this one would need to be on SscsCaseData